supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -86,7 +86,7 @@ Note that the IdM range and the DNA range match, but they are not interconnected
 [discrete]
 [[id-views]]
 ==== image:images/yes.png[yes] ID views (noun)
-*Description*: In Red Hat Enterprise Linux, ID views enable you to specify new values for POSIX user or group attributes, and to define on which client host or hosts the new values will apply. See examples of ID views usage:
+*Description*: In Red Hat Enterprise Linux, _ID views_ enable you to specify new values for POSIX user or group attributes, and to define on which client host or hosts the new values will apply. See examples of ID views usage:
```

Comment:
```suggestion
*Description*: In Red Hat Enterprise Linux, you can use _ID views_ to specify new values for POSIX users or group attributes, and to define on which client host or hosts the new values apply. See examples of ID views usage:
```
These suggestions are a bit nitpicky but
-  might reduce the anthropomorphic "enable you" that puts the technology in control
- Pluralizing "user" is a guess. Alternatively,  maybe it should  be "a POSIX user"
- I think we can use the present tense instead of the future

Reply:
Not nitpicky at all. A bit out of scope for this PR, but I might as well improve the wording while I'm making changes (as long as the technical meaning doesn't change).